crates/store/re_types/definitions/rerun/blueprint/archetypes/entity_behavior.fbs

```diff
@@ -0,0 +1,28 @@
+namespace rerun.blueprint.archetypes;
+
+/// General visualization behavior of an entity.
+///
+/// TODO(#6541): Fields of this archetype currently only have an effect when logged in the blueprint store.
+///
+/// \example archetypes/entity_behavior missing="cpp,rs"
```

Comment:
I believe you need to slap an `image="https://static.rerun.io/....` argument here for this snippet to be live.